quickstarts/microsoft.mobilenetwork/mobilenetwork-create-full-5gc-deployment/azuredeploy.json

```diff
@@ -314,7 +317,7 @@
         "servicePrecedence": 253,
         "pccRules": [
           {
-            "ruleName": "All_traffic",
```

Comment:
Reverted the Default PCC Rule Name to older format to have the consistence with the existing code. This will be addressed in back-end.